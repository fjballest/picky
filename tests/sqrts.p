/*
!../8.pick  -P sqrts.p && ../8.pam  p.out >[2=1]
 */
program sqrts;

procedure main()
	x: float;
{
	write(sqrt(4.0));
	writeeol();
	x = 4.0;
	write(sqrt(x));
	writeeol();
	x = -2.0;
	write(sqrt(x));
	writeeol();
}
