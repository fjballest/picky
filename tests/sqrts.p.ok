2.00000
2.00000
sqrts.p:15: run time violation: sys: fp: invalid operation fppc=0x89c0 status=0x81a1 pc=0x00005dac
pc=0x36, sp=0x50, fp=0x50
