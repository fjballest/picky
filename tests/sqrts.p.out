2.00000
2.00000
sqrts.p:15: run time violation: sys: fp: invalid operation fppc=0x8a58 status=0x81a1 pc=0x00005dc8
pc=0x36, sp=0x50, fp=0x50
